frontend/src/static/js/v_summary.js

```diff
@@ -278,6 +278,22 @@ window.vSummary = {
       return repo.location;
     },
 
+    getReportIssueLink(stackTrace) {
+      return 'https://github.com/reposense/RepoSense/issues/new?title=' +
+          encodeURI('Unexpected error with RepoSense version ') + window.app.repoSenseVersion + '&body=' +
+          encodeURI(stackTrace);
+    },
+
+    getReportIssueEmailAddress() {
+      return 'seer@comp.nus.edu.sg';
```

Comment:
this can be inlined into the `getReportIssueEmailLink` since its only used once.

Reply:
`getReportIssueEmailAddress` is also called in `index.pug` to get the displayed text for the link

Reply:
I see 👍